x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/trial_license_complete_tier/import_rules.ts

```diff
@@ -921,7 +921,7 @@ export default ({ getService }: FtrProviderContext): void => {
         });
       });
 
-      describe('migrate pre-8.0 action connector ids', () => {
+      describe('@skipInServerless migrate pre-8.0 action connector ids', () => {
```

Comment:
This test was not executed in the PR pipelines because it is in the `@skipInServerlessMKI` group. Now that it has been enabled, it is failing and needs to be fixed.

Reply:
Can you please skip this and another failing test in this Detection Rules Management owned file. I have created a [ticket](https://github.com/elastic/kibana/issues/188673) to fix and unskip them later.

Reply:
Created a ticket to fix and unskip: https://github.com/elastic/kibana/issues/188673